AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java

```diff
@@ -296,6 +302,39 @@ protected String getHumanReadableLoginErrorMessage(Exception exception) {
 
         return exception.getLocalizedMessage();
     }
+    
+    public void checkOrientation(int orientation) {
+        // if device is in horizontal mode then screen might not have enough space for ankidroid logo
+        // so we will invisible logo for horizontal mode only
+        if ((orientation == OrientationEventListener.ORIENTATION_UNKNOWN) || (orientation < 100) || (orientation > 280)) {
+            mAnkidroidLogo.setVisibility(View.GONE);
+        } else {
+            mAnkidroidLogo.setVisibility(View.VISIBLE);
+        }
+    }
+
+    public void initializeOrientationListner() {
+        myOrientationEventListener = new OrientationEventListener(this, SensorManager.SENSOR_DELAY_NORMAL) {
+            @Override
+            public void onOrientationChanged(int orientation) {
+                checkOrientation(orientation);
+            }
+        };
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        initializeOrientationListner();
```

Comment:
This should be in onCreate - you only need to initialize it once
onResume/onPause will just handle the enable/disable

```suggestion
        initializeOrientationListener();
```